### ISSUE TITLE
DatePicker dialog doesn't have accessible name

### ISSUE THREAD

**Issue by wemmyo** [jpmorganchase/salt-ds#5560]
### Latest version

- [x] I have tested the latest version

### Description

Raised issue from StackOverflow

Examples on site and storybook will need to be updated

### Steps to reproduce

Range Date Picker is missing title which can be seen when inspecting with chrome dev tools

### Package name(s)

Lab (@salt-ds/lab)

### Browser

- [x] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Microsoft Edge

### Operating system

- [x] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.

**Comment by mark-tate** [comment]
fixed in release @salt-ds/date-components 0.1.0
